src/pcapi/alembic/versions/20211104T135321_49f58959dbd4_recredit_table.py

```diff
@@ -0,0 +1,35 @@
+"""recredit-table
+"""
```

Comment:
Cette docstring est utilisée par Alembic dans `alembic history`. Je trouve pratique d'être explicite ici. Du genre :

```suggestion
"""Add Recredit model."""
```